src/App.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
 
+const todoList = [
+  {
+    id: 1,
+    title: "Read the book Road to React",
+  },
+  {
+    id: 2,
+    title: "See videos from Learn the App",
+  },
+  {
+    id: 3,
+    title: "Submit my assigment",
+  },
+  {
+    id: 4,
+    title: "Practice JavaScript",
+  },
+];
+
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
+    <div>
       <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
+        <h1>Todo List</h1>
       </header>
+        <ul>
+          {todoList.map(function (item) {
+            return <li>{item.id} {item.title}</li>;
```

Comment:
```suggestion
            return <li key={item.id}>{item.id} {item.title}</li>;
```
Sorry if you get two emails from this, I missed this bit. When using `.map` each returned child element needs a `key`, this is typically the item's id if it has one.